Sources/X509/Certificate.swift

```diff
@@ -209,6 +209,51 @@ public struct Certificate {
         self.signatureBytes = try DER.Serializer.serialized(element: ASN1BitString(self.signature))[...]
     }
 
+    /// Construct a certificate from constituent parts, signed by an issuer key.
+    ///
+    /// This API can be used to construct a ``Certificate`` directly, without an intermediary
+    /// Certificate Signing Request. The ``signature-swift.property`` for this certificate will be produced
+    /// automatically, using `issuerPrivateKey`.
```

Comment:
Let's add a section to the doc comment here that explicitly compares this to the other initializer, indicating that it defaults the `signatureAlgorithm` property based on the type of the key.

Reply:
Good idea.